src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -54,34 +59,46 @@ export class CheckBox extends React.Component<Props> {
     }
   };
 
+  private getSelectSize = (indeterminate: boolean, width: number, height: number): StyleType => {
```

Comment:
can be set in eva mappings?

Reply:
I guess no. We were trying to reach some consistency with nebular team